foodsaving/subscriptions/tests/test_fcm.py

```diff
@@ -19,30 +19,28 @@ def logger_warning_mock():
 
 
 @contextmanager
-def fcm_reload_without_config():
+def override_fcm_key(key=None):
```

Comment:
I suggested the `unset` option so you can actually unset it, passing the value as `None` is slightly different. But I think it's a tiny detail and nobody cares. So, I'm happy!